src/test/java/com/hotels/bdp/circustrain/bigquery/table/service/unpartitioned/UnpartitionedTableServiceTest.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.table.service.unpartitioned;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.junit.Test;
+
+import com.hotels.bdp.circustrain.bigquery.conversion.BigQueryToHiveTableConverter;
+
+public class UnpartitionedTableServiceTest {
+
+  private Table table = new BigQueryToHiveTableConverter().withTableName("foo").withDatabaseName("bar").convert();
+  private UnpartitionedTableService service = new UnpartitionedTableService(table);
+
+  @Test
+  public void getTable() {
+    UnpartitionedTableService service = new UnpartitionedTableService(table);
+    Table returned = service.getTable();
+    assertEquals(table, returned);
+    assertFalse(table == returned);
```

Comment:
assertNotEquals would give you a better error message if the match fails.

Reply:
I think something else is tested here because assertEquals(table, returned), but table != returned. I replaced the `assertFalse` with `assertNotSame` and it passed.

Reply:
Ah yes, the `==` check is just checking that they're not the same reference and that the service has returned some kind of wrapper around the table that is equivalent to the passed in `table` but isn't merely that returned directly. OK, well, leave it as is or replace this with `assertNotSame`, whichever you prefer, I think the latter is clearer as to the intent.